packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -237,6 +239,38 @@ example2 = {|x| x + 1}`,
       }),
     ],
   }),
+  maker.make({
+    name: "specification",
+    description: `Adds a specification to a value. This is useful for documenting how a value was calculated, or what it represents.`,
+    displaySection: "Tags",
+    definitions: [
+      makeDefinition(
+        [frWithTags(frAny({ genericName: "A" })), frSpecification],
+        frWithTags(frAny({ genericName: "A" })),
+        ([{ value, tags }, spec], reducer) => {
+          const showAs = spec.showAs && reducer.call(spec.showAs, [value]);
```

Comment:
Is this necessary?

Possibly I'm missing something, but couldn't you copy `spec.showAs` to the value tag, instead of calling it here?

Reply:
I tried this and it didn't work in your example, because the original `@showAs` is more strict in its current signature than the version in your example.

But then it seems surprising/inconsistent to relax that limitation in this specific situtation, because the limitation was intentional? If we want to allow a generic `@showAs(any) => any` then we should allow it everywhere.

(see also: my another comment on how `@showAs` is too powerful)